post-content.js

```diff
@@ -5,30 +5,58 @@ window._wpGutenbergPost = {
 	content: {
 		raw: [
 			'<!-- wp:core/heading -->',
-			'<h1>1.0 Is The Loneliest Number</h1>',
+			'<h1>Welcome to the Gutenberg Editor</h1>',
 			'<!-- /wp:core/heading -->',
 
 			'<!-- wp:core/text -->',
-			'<p>I imagine prior to the launch of the iPod, or the iPhone, there were teams saying the same thing: the copy + paste guys are <em>so close</em> to being ready and we know Walt Mossberg is going to ding us for this so let\'s just not ship to the manufacturers in China for just a few more weeks… The Apple teams were probably embarrassed. But <strong>if you\'re not embarrassed when you ship your first version you waited too long</strong>.</p>',
+			'<p>The goal of the new editor is to make adding rich content to WordPress simple and enjoyable. This whole post is composed of <em>pieces of content</em>—somewhat similar to Lego bricks—that you can move around and interact with. Move your cursor around and you\'ll notice the different blocks light up with outlines and arrows. Press the arrows to reposition blocks quickly, without fearing about losing things in the process of copying and pasting.</p>',
+			'<p>What you are reading now is a <strong>text</strong> <strong>block</strong>, the most basic block of all. A text block can have multiple paragraphs, if that\'s how you prefer to write your posts. But you can also split it by hitting enter twice. Once blocks are split they get their own controls to be moved freely around the post...</p>',
```

Comment:
Why are the individual words marked bold, instead of one tag encompassing both words?